bot/bot.py

```diff
@@ -0,0 +1,40 @@
+"""Utility class for interacting with Slack API"""
+import os
+from slackclient import SlackClient
+
+slack_token = os.environ["SLACK_API_TOKEN"]
```

Comment:
It would probably be a good idea to move this into the `__init__` function for the actual bot class. (Also, we would want to make this configurable at some point, e.g. via a yaml config file, but that doesn't need to be in this pull request.)

Reply:
Also try `os.environ.get('SLACK_API_TOKEN', <default value>)` to prevent key errors.